core/src/genesis.rs

```diff
@@ -88,31 +84,14 @@ async fn try_get_online_topology(
     network_topology: &Topology,
     network: Addr<IrohaNetwork>,
 ) -> Result<Topology> {
-    let (online_peers, offline_peers) =
+    let (online_peers, _offline_peers) =
         check_peers_status(this_peer_id, network_topology, network).await;
```

Comment:
change `check_peers_status` so that it returns only online peers

Reply:
okay